server/routes/auth/providers/google.ts

```diff
@@ -124,7 +128,7 @@ if (env.GOOGLE_CLIENT_ID && env.GOOGLE_CLIENT_SECRET) {
 
           return done(null, result.user, result);
         } catch (err) {
-          return done(err, null);
+          return done(PassportError(err, ctx), null);
```

Comment:
🥦 errors thrown out of authentication providers should be wrapped with `PassportError` to properly redirect

Reply:
(open to better names for this thing.)